### ISSUE TITLE
User admin page hangs

### ISSUE THREAD

**Issue by fbassas** [owncloud/core#25185]
### Steps to reproduce
1. Log in as admin
2. Select "Users" from right menu
3.

### Expected behaviour
List of users in few seconds

### Actual behaviour
Page nearly hangs giving browser message of "script taking too much time". you get list of users at last, but it takes a long time.

### Server configuration
**Operating system**: ubuntu 14.04

**Web server:** apache 2.4

**Database:** mysql 5.5

**PHP version:** 5.5.9

**ownCloud version:** (see ownCloud admin page) 9.0.2

**Updated from an older ownCloud or fresh install:**
updated from 8.2
**Where did you install ownCloud from:**
repositories
**Signing status (ownCloud 9.0 and above):**

```
Login as admin user into your ownCloud and access 
http://example.com/index.php/settings/integrity/failed 
paste the results here.
```

**List of activated apps:** documents,calendars,contacts

```
If you have access to your command line run e.g.:
sudo -u www-data php occ app:list
from within your ownCloud installation folder
Enabled:
  - activity: 2.2.1
  - bookmarks: 0.7
  - calendar: 1.2.2
  - comments: 0.2
  - contacts: 1.3.1.0
  - dav: 0.1.6
  - documents: 0.12.0
  - federatedfilesharing: 0.1.0
  - federation: 0.0.4
  - files: 1.4.4
  - files_pdfviewer: 0.8.1
  - files_sharing: 0.9.1
  - files_texteditor: 2.1
  - files_trashbin: 0.8.0
  - files_versions: 1.2.0
  - files_videoplayer: 0.9.8
  - firstrunwizard: 1.1
  - gallery: 14.5.0
  - notifications: 0.2.3
  - provisioning_api: 0.4.1
  - systemtags: 0.2
  - templateeditor: 0.1
  - updatenotification: 0.1.0
  - user_ldap: 0.8.0
Disabled:
  - encryption
  - extendedtrash
  - external
  - files_external
  - ocusagecharts
  - search_lucene
  - user_external


**The content of config/config.php:**

```
If you have access to your command line run e.g.:
sudo -u www-data php occ config:list system
from within your ownCloud installation folder

or 

Insert your config.php content here
(Without the database password, passwordsalt and secret)
$CONFIG = array (
  'instanceid' => 'ocde02dfcbf8',
  'passwordsalt' => '********************************************',
  'datadirectory' => '/var/www/owncloud/data',
  'dbtype' => 'mysql',
  'version' => '9.0.2.2',
  'dbname' => 'owncloud',
  'dbhost' => 'localhost',
  'dbtableprefix' => 'oc_',
  'dbuser' => 'admin',
  'dbpassword' => '***********',
  'installed' => true,
  'ldapUserCleanupInterval' => 51,
  'theme' => 'sict',
  'maintenance' => false,
  'trusted_domains' => 
  array (
    0 => 'nuvol.terrassa.upc.edu',
  ),
  'mail_smtpmode' => 'smtp',
  'mail_smtphost' => 'relay-ct.upc.edu',
  'mail_smtpport' => '25',
  'mail_smtptimeout' => 10,
  'memcache.distributed' => '\\OC\\Memcache\\Memcached',
  'memcache.local' => '\\OC\\Memcache\\Memcached',
  'memcached_servers' => 
  array (
    0 => 
    array (
      0 => 'localhost',
      1 => 11211,
    ),
  ),
  'share_folder' => '/Shared',
  'overwritewebroot' => '/owncloud',
  'loglevel' => 2,
  'log_rotate_size' => 104857600,
  'preview_libreoffice_path' => '/usr/bin/libreoffice',
  'mail_from_address' => 'owncloud.terrassa',
  'mail_domain' => 'ct.upc.edu',
  'secret' => 'b76126bb72f1ab011163a2d0eb15ff9f0ac31f82405b1826c4421ebf57cfe0c42c828d5af89374d61b9a942bf911b6e8',
  'overwrite.cli.url' => '/owncloud',
  'trashbin_retention_obligation' => 'auto',
  'activity_expire_days' => '180',
  'updatechecker' => false,
);

**Are you using external storage, if yes which one:** local/smb/sftp/...
NO
**Are you using encryption:** yes/no
NO
**Are you using an external user-backend, if yes which one:** LDAP/ActiveDirectory/Webdav/...
NO
#### LDAP configuration (delete this part if not used)

```
With access to your command line run e.g.:
sudo -u www-data php occ ldap:show-config
from within your ownCloud installation folder
| hasMemberOfFilterSupport      |                                                                                                        |
| hasPagedResultSupport         |                                                                                                        |
| homeFolderNamingRule          |                                                                                                        |
| lastJpegPhotoLookup           | 0                                                                                                      |
| ldapAgentName                 | cn=smbldap-tools,ou=DSA,dc=ct,dc=upc,dc=es                                                             |
| ldapAgentPassword             | ***                                                                                                    |
| ldapAttributesForGroupSearch  | cn                                                                                                     |
| ldapAttributesForUserSearch   | uid;displayname                                                                                        |
| ldapBackupHost                | ldaps://ldapsec-ct.upc.edu                                                                             |
| ldapBackupPort                | 636                                                                                                    |
| ldapBase                      | dc=ct,dc=upc,dc=es                                                                                     |
| ldapBaseGroups                | ou=Groups,dc=ct,dc=upc,dc=es                                                                           |
| ldapBaseUsers                 | ou=Users,dc=ct,dc=upc,dc=es                                                                            |
| ldapCacheTTL                  | 600                                                                                                    |
| ldapConfigurationActive       | 1                                                                                                      |
| ldapDynamicGroupMemberURL     |                                                                                                        |
| ldapEmailAttribute            | mail                                                                                                   |
| ldapExperiencedAdmin          | 0                                                                                                      |
| ldapExpertUUIDGroupAttr       |                                                                                                        |
| ldapExpertUUIDUserAttr        |                                                                                                        |
| ldapExpertUsernameAttr        |                                                                                                        |
| ldapGroupDisplayName          | cn                                                                                                     |
| ldapGroupFilter               | (&(objectClass=ownCloudGroup)(ownCloudEnabled=1))                                                      |
| ldapGroupFilterGroups         | cd6;plataforma;sbasics;sict;sict220;sict320;sict370;sictserveis;som;som170;telematica;telesom;ugct;ugt |
| ldapGroupFilterMode           | 1                                                                                                      |
| ldapGroupFilterObjectclass    |                                                                                                        |
| ldapGroupMemberAssocAttr      | memberUid                                                                                              |
| ldapHost                      | ldaps://ldap-ct.upc.edu                                                                                |
| ldapIgnoreNamingRules         |                                                                                                        |
| ldapLoginFilter               | (&(objectclass=inetOrgPerson)(|(uid=%uid)(|(uid=%uid))))                                               |
| ldapLoginFilterAttributes     | uid                                                                                                    |
| ldapLoginFilterEmail          | 0                                                                                                      |
| ldapLoginFilterMode           | 1                                                                                                      |
| ldapLoginFilterUsername       | 1                                                                                                      |
| ldapNestedGroups              | 0                                                                                                      |
| ldapOverrideMainServer        | 0                                                                                                      |
| ldapPagingSize                | 500                                                                                                    |
| ldapPort                      | 636                                                                                                    |
| ldapQuotaAttribute            | ownCloudQuota                                                                                          |
| ldapQuotaDefault              | 2 G                                                                                                    |
| ldapTLS                       | 0                                                                                                      |
| ldapUserDisplayName           | uid                                                                                                    |
| ldapUserDisplayName2          |                                                                                                        |
| ldapUserFilter                | (&(objectclass=inetOrgPerson)(!(sambaAcctFlags=[DUX]))(!(gidNumber=1079)))                             |
| ldapUserFilterGroups          |                                                                                                        |
| ldapUserFilterMode            | 1                                                                                                      |
| ldapUserFilterObjectclass     | inetOrgPerson                                                                                          |
| ldapUuidGroupAttribute        | auto                                                                                                   |
| ldapUuidUserAttribute         | auto                                                                                                   |
| turnOffCertCheck              | 0                                                                                                      |
| useMemberOfToDetectMembership | 1                                                                                                      |


### Client configuration
**Browser:** firefox

**Operating system:** ubuntu

```

**Comment by RealRancor** [comment]
Probably a duplicate of https://github.com/owncloud/core/issues/24734 which gets fixed in 9.0.3

**Comment by fbassas** [comment]
It seems so. I have applied suggested workaround.
Thanks for your help.
